product_docs/docs/biganimal/release/administering_cluster/04_customizing_policy_ definitions.mdx

```diff
@@ -2,13 +2,14 @@
 title: "Customizing Azure policy definitions"
 ---
 
-Microsoft Defender for Cloud (which now includes Azure Secure Center and Azure Defender) analyzes the configurations of your Azure resources to identify potential vulnerabilities. Microsoft Defender for Cloud uses Azure Policy to help you monitor, identify, and remediate non-compliant resources. 
+Azure Policies help you monitor, identify, and remediate non-compliant resources. Azure automatically assigns a default set of policies to each subscription. If required by your organization, you can further customize these default Azure policies to match BigAnimal's resource configurations. 
 
-Azure automatically assigns a default set of policies to each subscription belonging to your organization. You can further customize these default Azure policies based on how resources are configured in BigAnimal. See [Customizable policy definition parameters](#customizable-policy-definition-parameters) for a recommended list of policy definition parameters that you can customize.
+!!! Note
+    BigAnimal does not customize your Azure policies to prevent conflicts with external workloads.
 
 ### Customize default policy definitions in Azure
 
-BigAnimal does not customize your Azure policies to prevent conflicts with external workloads. To customize Azure policies, you must manually perform the following steps in each of your Azure subscriptions:
+Manually perform the following steps in each of your Azure subscriptions:
```

Comment:
This should read something like, "in your BigAnimal-enabled Azure subscriptions", yes?

Reply:
Fixed.